chat: rename online status to presence (#2630)
ably/docs: src/pages/docs/chat/rooms/presence.mdx

---
title: Presence
meta_description: "Use presence to see which users are online and their user status."
---

> The presence feature of a chat room allows online users to advertise to others that they are online, which can be used to build online status indicators to display who is online in a chat room. Users can also share additional information for others to use, including avatar URLs or a custom status.

Reviewer: ```suggestion
The presence feature of a chat room enables online users to advertise to others that they are online. Use this feature to build online status indicators to display who is online in a chat room. Users can also share additional information such as avatar URLs or custom statuses.
```
